src/main/java/ev3dev/sensors/ev3/EV3IRSensor.java

```
@@ -216,12 +162,13 @@ public int getRemoteCommand(int chan) {
      * @param len the number of commands to store.
      */
     public void getRemoteCommands(byte[] cmds, int offset, int len) {
-        switchMode(MODE_REMOTE, SWITCH_DELAY);
+        // TODO this should read multiple commands, but we probably cannot easily wait for new ones
```

Hmmm, we could throw an exception here.